hack/run-locally.sh

```diff
@@ -152,9 +154,20 @@ function create_cluster() {
     trap "trap - TERM && kill -- -$$" INT TERM EXIT
 }
 
+function detect_platform() {
+    if [[ -n "$(grep '\"type\": \"libvirt_' ${CLUSTER_DIR}/terraform.tfstate)" ]]; then
+        PLATFORM=libvirt
+    elif [[ -n "$(grep '\"type\": \"aws_' ${CLUSTER_DIR}/terraform.tfstate)" ]]; then
+        PLATFORM=aws
+    else
+        echo "could not detect install platform from terraform.tfstate" >&2
+        exit 1
+    fi
+}
+
 # Fix up the AWS firewall so network plugin components can talk to each other
 function open_aws_ports() {
-    if [[ "${NETWORK_PLUGIN}" == "OVNKubernetes" ]]; then
+    if [[ "${NETWORK_PLUGIN}" == "OVNKubernetes" && "${PLATFORM}" == "aws" ]]; then
```

Comment:
I was assuming that if multi-platform run-locally.sh support arrived before open-ovn-ports.sh went away that we'd add support for other platforms to open-ovn-ports.sh. Is there no firewalling in libvirt?

Reply:
@danwinship there's whatever firewalling RHCOS does, which appears to be literally nothing. Like, all it has are nat+filter tables with default chain policies of ACCEPT. So I guess if that ever changes or it ever starts running firewalld we'd need to update this.